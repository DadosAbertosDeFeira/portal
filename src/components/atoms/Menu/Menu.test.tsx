import { render } from "@testing-library/react";

import { Menu, type MenuProps } from ".";

describe("Menu", () => {
  const makeSut = ({ ...props }: Partial<MenuProps>) => {
    return render(<Menu {...props} />);
  };

  it("renders correctly", () => {
    const { asFragment } = makeSut({});

    expect(asFragment).toMatchSnapshot();
  });

  it("should render header correctly", () => {
    const { asFragment } = makeSut({ header: <p>Header</p> });

    expect(asFragment).toMatchSnapshot();
  });

  it("should render footer correctly", () => {
    const { asFragment } = makeSut({ footer: <p>Footer</p> });

    expect(asFragment).toMatchSnapshot();
  });
});
