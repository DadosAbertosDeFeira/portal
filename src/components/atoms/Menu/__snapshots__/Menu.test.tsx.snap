// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu renders correctly 1`] = `[Function]`;
